src/main/java/org/spongepowered/common/mixin/core/entity/item/FireworkRocketEntityMixin.java

```diff
@@ -111,7 +111,7 @@
     }
 
     @SuppressWarnings("deprecation")
-    @Redirect(method = "onUpdate", at = @At(value = "INVOKE", target = "Lnet/minecraft/world/World;setEntityState"
+    @Redirect(method = "tick()V", at = @At(value = "INVOKE", target = "Lnet/minecraft/world/World;setEntityState"
                                                                        + "(Lnet/minecraft/entity/Entity;B)V"))
     private void onExplode(final World world, final Entity self, final byte state) {
```

Comment:
Fix the method prefix and method name, `onExplode` makes little to no sense of what it's doing.